x-pack/plugins/apm/common/agent_configuration/setting_definitions/general_settings.ts

```diff
@@ -527,6 +543,7 @@ export const generalSettings: RawSettingDefinition[] = [
           'By default, the agent will sample every transaction (e.g. request to your service). To reduce overhead and storage requirements, you can set the sample rate to a value between 0.0 and 1.0. We still record overall time and the result for unsampled transactions, but not context information, labels, or spans.',
       }
     ),
+    excludeAgents: ['android/java'],
```

Comment:
We want the same for `ios/swift` agent, no?

Reply:
That's right! Let me double check with @bryce-b in case he had other plans for now, otherwise I can add it here right away.

Reply:
It's added now 👍